release-1.8/release_notes_draft.md

```diff
@@ -478,16 +478,50 @@ kind.
 
 ### API Machinery
 
+
+#### kube-apiserver
+* Fixes an issue with APIService auto-registration affecting rolling HA apiserver restarts that add or remove API groups being served.([#51921](https://github.com/kubernetes/kubernetes/pull/51921))
```

Comment:
Fixes an issue with API Service auto-registration. This issue affected restarting the HA API server that add or remove API groups being served.

Reply:
no, `APIService` is the object kind